### ISSUE TITLE
Actions are not Updating App State. Strings arent adding to Lists in DataTypes.  

### ISSUE THREAD

**Issue by Coopertron1** [FlutterFlow/flutterflow-issues#3133]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

It seems like the latest FlutterFlow update has had an impact on updating or adding strings to app state / data type lists. 
<img width="304" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/172756480/e7c98c4d-4771-4f67-8fea-b63102d93148">

Before todays FF update we didnt have an issue with adding stings to lists in the app state. 

### Expected Behavior

When I toggle the widget on a string should add to the App State ' FatigueCheck'

### Steps to Reproduce

1) Run a Test Environment 
2) Click begin Fatigue Check (https://app.flutterflow.io/project/fatigue-tech-app-6fpzwq?tab=widgetTree&page=FatigueCheckAssessment)  
3) Select multiple Tasks where is says 'Describe your current task?'
4) Watch the Debug Panel for strings adding into FatigueCheck app state. Specifically the CurrentTask list. 
<img width="190" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/172756480/9f38523a-8f86-4a2d-8eae-239ed40d1452">

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4CkvLq2492xdtc+KXqdfowqyQsKUg4RYEnj+xhUUsaHZPQDLN/ds/AP1NuON+9YHxDfU2Mp14B+c77ivDLU+4rHwueG5hbyKh+YQvyTTmneKmTF7ueREclL9pNBX663MONuBN+KrBaWXsu72WmCd6vUT3cGOPGOGo8Sq/LZO4=

### Visual documentation

https://github.com/FlutterFlow/flutterflow-issues/assets/172756480/94e896a0-4960-4905-8e1a-591085e2242a

Below is the action on the Fatigue Check Assessment page. This is happening across the app though. 
<img width="604" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/172756480/55746089-9b58-4aad-ba90-af5a6f9e37cc">

### Environment

```markdown
- FlutterFlow version: 4.1.59
- Platform: Web, Android, IOS and test environment 
- Browser name and version:
- Operating system and version affected:
```

**Comment by vitoras** [comment]
Something big was clearly broken in the latest FF update, lots of bugs being reported from everyone.

**Comment by paulperez-dev** [comment]
Hi @Coopertron1! Sorry you are facing this issue. I tried to replicate the bug but the state is being updated. Notice the debug panel is still in beta. Can you please try to replicate the issue in a blank project and share de bug report code again?

**Comment by Coopertron1** [comment]
Hi @paulperez-dev. 
Did you go through the FatigueCheck process? Did you see that none of the Tasks, Causes or Controls get added to the report at the end of the process. They use to add to the App State but now they are fallen off. The Update App State List Action across multiple projects. I wasn't only looking at the bug report. That was just a good way to show you that it wasnt updating list.   

<img width="220" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/172756480/08012cc3-6d52-439b-b9de-d816ac25b09a">

**Comment by ready4sea** [comment]
Unfortunately, this is probably the same as the confirmed bug https://github.com/FlutterFlow/flutterflow-issues/issues/3111, which has been open for 6 days, is causing major issues, but which has not been addressed and has no ETA.

**Comment by JP-BLURP** [comment]
I have the same issue :\ been going crazy trying to work around it

**Comment by paulperez-dev** [comment]
Hi @Coopertron1! Sorry for the late response. Can you please confirm the following:

1. Is this the same issue that was reported [here](https://github.com/FlutterFlow/flutterflow-issues/issues/3111)?
2. Is this issue still present in the newest version?

**Comment by Coopertron1** [comment]
Hi @paulperez-dev, 

I have just tested it and my issue has been resolved. It had been resolved 1.5 weeks ago I think. Yes, this was the same issue as #3111. Thanks for following up.

**Comment by paulperez-dev** [comment]
Thanks for the confirmation @Coopertron1! Closing this!
